src/scss/base/_grid.scss

```diff
@@ -3,16 +3,12 @@
   column-gap: 5px;
   display: grid;
   grid-template-columns: repeat(12, 1fr);
-  max-width: 360px;
-  width: 100%;
-  //Grid Desktop (No defined yet)
-}
-
-.container {
```

Comment:
Do not remove this class because it is being used in several components

Reply:
@jsebax this class it's no necessary to have it since the container must be set with the properties of the class .grid

Reply:
Please leave the class `.container` for now. There are other components using this. The refactor can be done in other PR, if needed.

Reply:
Use the `.grid` class in your component, but keep the `.container` class so that you don't affect other components that much. The refactor is quite large because the design in both, mobile and desktop, should work as it was before.